client/js/widgets/widget.js

```diff
@@ -193,6 +201,24 @@ export class Widget extends StateManaged {
       }
     }
 
+    if(this.activateAnimation) {
+      this.animateProperties().forEach((prop)=>{
+        if(prop != null) {
+          const rule = (typeof prop == 'object')? prop : { property: prop };
+          if(delta[rule.property] !== undefined) {
+            if(rule.className == null)
+              rule.className = `animate_${escapeID(rule.property)}`;
+            if(typeof rule.duration != 'number')
+              rule.duration = 1000;
+            if(this.animateTimeouts[rule.className])
+              clearTimeout(this.animateTimeouts[rule.className]);
+            this.domElement.classList.add(rule.className);
+            this.animateTimeouts[rule.className] = setTimeout(()=>this.domElement.classList.remove(rule.className),rule.duration);
```

Comment:
It is possible that classes that were not initially set by `animatePropertyChange` rules could be removed by them. Probably should filter the remove action.

Reply:
This is now fixed, also fixed that temporary classes will not be removed when other classes are changed.